course_discovery/apps/edx_elasticsearch_dsl_extensions/tests/test_boosting.py

```diff
@@ -113,17 +114,21 @@ def test_expired_paid_seat_penalized(self):
             with patch.object(CourseRun, 'get_paid_seat_enrollment_end', return_value=past):
                 penalized_run = self.build_normalized_course_run(title='penalized')
 
-        search_results = SearchQuerySet().models(CourseRun).all()
+        search_results = CourseRunDocument.search().query(ESDSLQ('match_all')).execute()
         assert len(search_results) == 2
 
         assert [promoted_run.title, penalized_run.title] == [hit.title for hit in search_results]
-        assert search_results[0].score > search_results[1].score
+        assert search_results[0].meta['score'] > search_results[1].meta['score']
 
         # Verify that this result has a negative score. Course runs with expired,
         # paid seats are penalized by having a relatively large value subtracted
         # from their relevance score. In this test case, the result should be a
         # negative relevance score.
-        assert 0 > search_results[1].score
+        # FIXME: The test case bellow it not relevant now cause In es7 negative values do not supported.
+        # Need think how to handle the filter.
+        # See the link, probably it could be useful to make a decision
+        # https://github.com/elastic/elasticsearch/issues/24910
+        # assert 0 == search_results[1].meta['score']
```

Comment:
Did this get resolved one way or the other? Can just delete this block if we don't need it.

Reply:
I totally forgot about this place. 
This was done before we were looking for a solution to how to handle the `negative case`. We applied inversion here as you remember and cases are the same but in reverse. I will not delete it, but simply change the test to fit the new conditions.

Reply:
Pushed a commit about it https://github.com/raccoongang/course-discovery/pull/12/commits/3fc5caf0452c27d1f113185061f04771bb8288bb